src/service/iot_config.proto

```diff
@@ -438,6 +485,11 @@ service gateway {
       returns (gateway_region_params_res_v1);
   // Get H3 Location for a gateway (auth admin only)
   rpc location(gateway_location_req_v1) returns (gateway_location_res_v1);
+  // Get info for the specified gateway
+  rpc info(gateway_info_req_v1) returns (gateway_info_res_v1);
+  // Get a stream of gateway info
+  rpc info_stream(gateway_info_stream_req_v1)
+      returns (stream gateway_info_stream_res_v1);
```

Comment:
do we want or is there any advantage in having the info stream return a different/reduced message type than the batches of the full `gateway_info` when it's only being consumed to build the transmit scaling map and that only needs the `location` field? especially since we just changed the blockchain-node implementation to return an empty region_params field for this request/response type?
i _believe_ we could change this to just return a stream of batches of `gateway_location_res_v1` if all we expect to need is to maintain compatibility with how the verifier is using the current node but we could also have a separate `gateway_metadata_res_v1` that just includes the location and other fields we might need for _all_ gateways

Reply:
so i am hoping to be able to use the stream response to prewarm the gateway cache in the verifier.  That will save considerable time after a reboot.  Currently we await ~5 mins to stream the gateways to the density scaler and then ~20mins to complete the first batch of beacons and witnesses.  Prewarming the cache will reduce this time significantly

Reply:
Does that include the region params or can those still be omitted for that cache? If you need those region params then we’ll likely still have to solve for some significant memory pressure on the config service without tying the cached region params on _that_ side into the info stream

Reply:
Given region params dont change all that often and there is a small set of these, we can probably do without and pull those from a local verifier cache based on the gateway.region field.

Reply:
so we can likely drop the region params from the gateway info struct

Reply:
I suspect an RPC to the config service to return the region params for a specified region will be the best fit here.  Will confirm after working through some things on the verifier side

Reply:
I see there is an existing `region_params` RPC.  That works for my needs

Reply:
that one may not be the best bet if you want to load the params once and cache them since that rpc takes the pubkey of a hotspot in the request (and the hotspot’s “default” region) and looks up the pubkey on-chain to return the asserted region’s params

Reply:
Added this: 4212e57086fb29c05e71e9c0e48bc2c034f9d877 to return region params for the specified region.  Its very close to the GatewayRegionParamsReqV1, is that RPC actually in use atm ?

Reply:
I'm not sure I like the way this is coming together right now.

Reply:
in what sense? specifics would be helpful

Reply:
in this case i would expect the signing key for the verifier, or any other oracle, to have its pubkey registered to the config service (perhaps under a new key type of "oracle" as opposed to the "administrator" or "packet router" types) using the admin service's `add_key` rpc. for any of these methods we want to truly authorize we have to save the key in the config service somewhere and retrieve it to check the request is authorized anyway so sending the pubkey in the request doesn't seem to add any value to me.

Reply:
having requests coming from the gateways themselves include their pubkeys for signature validation make sense on the other hand since there's enough of those to make persisting them infeasible and we're not using those as an authorization gate to restrict access to the data